### DISCUSSION TITLE
[CSS] styling for pseudo classes

### DISCUSSION THREAD

**Discussion by seedy** [resend/react-email#1086] [Q&A]
Is it possible yet to write styles for pseudo classes ?

My use case : I want to customize hover, focus, focus-visible, active styles on my buttons, icon buttons and links.

![Screenshot 2023-12-07 at 17 55 10](https://github.com/resendlabs/react-email/assets/3367393/cec5d98d-9490-43e8-8fb2-bca9746c6646)

**Comment by WillTaylor22** [comment]
+1 !
